src/examples/java/com/fingerprint/example/FunctionalTests.java

```diff
@@ -16,14 +18,18 @@ public static void main(String... args) {
         FingerprintApi api = new FingerprintApi(client);
 
         try {
-            api.getEvent(FPJS_REQUEST_ID);
+            EventResponse response = api.getEvent(FPJS_REQUEST_ID);
+
+            System.out.println("Event Response: " + response.toString());
         } catch (ApiException e) {
             System.err.println("Exception when calling FingerprintApi.getEvent:" + e.getMessage());
             System.exit(1);
         }
 
         try {
-            api.getVisits(FPJS_VISITOR_ID, null, null, null, null, null);
+            Response response = api.getVisits(FPJS_VISITOR_ID, null, null, null, null, null);
+
+            System.out.println("Visits Response: " + response.toString());
```

Comment:
Why do we need this console outputs for?

Reply:
Since it's an example, users might try to run it locally, so I think producing some output that they can see is good.